[T112323663] Build a graph visualiser to display PyTorch models with Dynamic Tracer

petrovs12 opened the pull request:
Test Plan:
Added functionality, exporting a dynamic trace fx object to networkx graph, and
a visuailzation script in dash-cytoscape with an web app for exploring said graph, given a dynamic trace results object.

Reviewed By: simonhollis

Differential Revision: D40528697

petrovs12 reviewed (commented):
Hi, this is part of an internal diff, would it be possible for someone toapprove so I can land it, please?

simonhollis reviewed (approved):
The trivial colour addition to the `graph_drawer` is fine.

petrovs12 committed:
[T112323663] Build a graph visualiser to display PyTorch models with Dynamic Tracer (#92899)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/92899

Test Plan:
Added functionality, exporting a dynamic trace fx object to networkx graph, and
a visuailzation script in dash-cytoscape with an web app for exploring said graph, given a dynamic trace results object.

Reviewed By: simonhollis

Differential Revision: D40528697

fbshipit-source-id: 2b689dc87dc6f2761ea8089926f0785bbc1a6c06
